test/foundry/SignatureChecker.t.sol

```diff
@@ -1,9 +1,22 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.14;
 
-import {SignatureChecker, ISignatureChecker} from "../../contracts/SignatureChecker.sol";
+import {SignatureChecker} from "../../contracts/SignatureChecker.sol";
 import {ERC1271Contract} from "./utils/ERC1271Contract.sol";
 import {TestHelpers} from "./utils/TestHelpers.sol";
+import "../../contracts/errors/SignatureCheckerErrors.sol";
+
+contract PublicSignatureChecker {
```

Comment:
There is a duplicate of `test/foundry/utils/PublicSignatureChecker.sol`

Reply:
oops

Reply:
Adjusted